R/SpatialExperiment.R

```diff
@@ -271,13 +272,15 @@ SpatialExperiment <- function(...,
     msg <- function(.) message(sprintf(paste(                
         "both '%s' and '%sNames'  have been supplied; using '%s'.",
         "Set either to NULL to suppress this message"), ., ., .))
+        ## ^ this message 
+        ## is printed also in case the spatialCoords and CoordsNames are not 
+        ## both provided!! 
```

Comment:
Can you give an example? Otherwise remove the comment. It's only printed when I pass both `spatialData/Coords` and `-Names`, but not if only one of them is provided, which is how it's intended to work.

Reply:
I'll remove it for now